driver.sh

```diff
@@ -51,6 +51,18 @@ setup_variables() {
       qemu_cmdline=( -drive "file=images/x86_64/rootfs.ext4,format=raw,if=ide"
                      -append "console=ttyS0 root=/dev/sda" ) ;;
 
+    "ppc32")
+      config=ppc44x_defconfig
+      image_name=zImage
+      qemu="qemu-system-ppc"
+      qemu_ram=128m
+      qemu_cmdline=( -machine bamboo
+                     -dtb "arch/powerpc/boot/dts/bamboo.dtb"
+                     -append "console=ttyS0"
+                     -initrd "images/ppc32/rootfs.cpio" )
```

Comment:
Hmm shouldn't `-no-reboot` be added to this too?

Reply:
Yes. I have it here locally... Good catch!